frontend/src/stories/Drawer/Drawer.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import * as BPCore from '@blueprintjs/core';
+
+export function Drawer({isOpen, title, onClose, children, position, size}) {
+	const positionValue = (
+		function(position) {
+			switch(position) {
+				case "TOP":
+				case "Top":
+				case "top":
+				case "t":
+					return BPCore.Position.TOP;
+				case "BOTTOM":
+				case "Bottom":
+				case "bottom":
+				case "b":
+					return BPCore.Position.BOTTOM;
+				case "LEFT":
+				case "Left":
+				case "left":
+				case "l":
+					return BPCore.Position.LEFT;
+				default:
+					return BPCore.Position.RIGHT;
+			}
+		}
+	)(position);
```

Comment:
I haven't bothered with this type of thing.  Feel unnecessary and encourages sloppy values.

I suspect BPCore.Position.TOP has a concrete string value "top" or whatever.  Let's just use that.

Reply:
Yes, they do. I hadn't realised. Change just added.

I also realise that I neglected to add prop types, so those have been added too.